canvas_modules/harness/package.json

```diff
@@ -34,15 +36,21 @@
     "@babel/polyfill": "7.12.1",
     "@babel/preset-env": "7.24.3",
     "@babel/preset-react": "7.24.1",
+    "@babel/register": "7.24.6",
     "@carbon/charts": "1.15.3",
     "@carbon/charts-react": "1.15.3",
-    "@carbon/react": "^1.52.0",
+    "@carbon/react": "1.52.0",
+    "@cypress/code-coverage": "3.12.39",
     "@elyra/canvas": "file:../common-canvas",
+    "@istanbuljs/nyc-config-babel": "3.0.0",
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.11",
     "ajv": "8.12.0",
     "autoprefixer": "10.4.19",
+    "babel-core": "6.26.3",
     "babel-loader": "9.1.3",
+    "babel-plugin-istanbul": "6.1.1",
     "babel-plugin-lodash": "3.3.4",
+    "babel-preset-env": "1.7.0",
```

Comment:
Why do we need both babel-preset-env and @babel/preset-env?  Shouldn't we be using only `@babel` modules?